engine/loader.asm

```diff
@@ -1087,7 +1118,9 @@ ProgressDone@
 *
 Ldr_Unload_Level
             * 1. Start by freeing all of the 8k blocks allocated
-            lda         #VH_LVLOBJCODE          * free the level/object code page
+            lda         #VH_LVLOBJCODE1         * free the level/object code page
```

Comment:
This will always free exactly 2 blocks, and probably not operate properly if the level has 1 or more than 2 blocks. I think we need a loop here.

Reply:
Yes, agree 100%

Reply:
Fixed